NEWS.rst

```diff
@@ -59,13 +59,17 @@ from which the enzyme object was created, and a `uri` property with a canonical
 Additionally, a number of small bugs and typos have been fixed with additions
 to the test suite.
 
+Add new ``gc_content`` function in ``SeqUtils`` and marks ``GC`` for future
+deprecation.
+
 Many thanks to the Biopython developers and community for making this release
 possible, especially the following contributors:
 
 - Andrius Merkys
 - Arup Ghosh (first contribution)
 - Aziz Khan
 - Alex Morehead
+- Caio Fontes
```

Comment:
This is your first credited contribution, right?

Reply:
In NEWS.rst yes, but I already had my name in CONTRIB.rst due to #3524

Reply:
Ah - I didn't check, normally we'd have put your name in NEWS alongside adding you to CONTRIB.